crates/cairo-lint-core/tests/test_files/double_comparison/double_comparison

```diff
@@ -0,0 +1,401 @@
+//! > contradictory equal and less than
```

Comment:
maybe i missed but can you also add a test where you invert x and y, for example. `x >= y && x y >= x`